test/features/idv/idv_service_test.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:didpay/features/idv/idv_service.dart';
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:http/http.dart' as http;
+
+import 'package:mocktail/mocktail.dart';
+import 'package:web5/web5.dart';
+
+import '../../helpers/mocks.dart';
+
+void main() {
+  late MockHttpClient mockHttpClient;
+  late IdvService subject;
+
+  setUp(() {
+    mockHttpClient = MockHttpClient();
+    subject = IdvService(httpClient: mockHttpClient);
+  });
+
+  group('IdvService', () {
+    test('should get IDV request', () async {
+      // arrange
```

Comment:
ubernit, but you can probably just think `arrange, act, assert` in your head vs. having comments in code for it :)